public/keymaps/c/coseyfannitutti_romeo_default.json

```
@@ -0,0 +1,20 @@
+{
+  "keyboard": "coseyfannitutti/romeo",
+  "keymap": "default_2a31fbf",
+  "commit": "2a31fbf9a6970ed425c2331f25fb7f92648ffcf1",
+  "layout": "LAYOUT_all",
+  "layers": [
+    [
+      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_BSPC",
+      "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_QUOT",
+      "KC_LSFT", "KC_SLSH", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_RSFT",
+      "KC_LCTL", "KC_LGUI", "KC_LALT",                       "KC_SPC",  "KC_SPC",  "KC_SPC",             "KC_RALT", "MO(1)",   "KC_RCTL",
+    ],
+    [
+      "KC_ESC",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_RESET",
+      "KC_VOLU", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_LEFT", "KC_DOWN", "KC_UP",   "KC_RIGHT","KC_ENT",
+      "KC_LSFT",            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_MUTE", "KC_EQL",  "KC_SLSH", "KC_TRNS",
+      "KC_TRNS", "KC_TRNS", "KC_TRNS",                       "KC_TRNS", "KC_TRNS", "KC_TRNS",            "KC_TRNS", "KC_TRNS", "KC_TRNS",
```

- `KC_RESET` &rarr; `RESET`
- `KC_RIGHT` &rarr; `KC_RGHT` (has rendering implications in the UI)
- Fix the missing key between Left Shift and Z.
- Deletes an extra comma at the end.

```suggestion
      "KC_ESC",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "RESET",
      "KC_VOLU", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_LEFT", "KC_DOWN", "KC_UP",   "KC_RGHT", "KC_ENT",
      "KC_LSFT", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_MUTE", "KC_EQL",  "KC_SLSH", "KC_TRNS",
      "KC_TRNS", "KC_TRNS", "KC_TRNS",                       "KC_TRNS", "KC_TRNS", "KC_TRNS",            "KC_TRNS", "KC_TRNS", "KC_TRNS"
```